content/v0.0.1/index.md

```diff
@@ -6,3 +6,290 @@ aliases: ["/en/"]
 # Conventional Logs v0.0.1
 
 ## Summary
+
+The conventional logs specification is an opinionated convention
+aiming to set an industry standard to ensure search-ability
+and enhance security in logs.
+
+Each log entry could be encoded in different formats, `json`, `xml`, or even a custom one.
+Most of the examples in this page will be written as `json`.
+
+A basic log entry should be structured as follows:
+
+---
+
+```
+{
+  "severity": "DEBUG",
+  "scope": "web-app",
+  "message": "could not save the user",
+  "timestamp": "2022-05-14T14:16:15+00:00"
+}
+```
+
+---
+
+A minimal log entry is composed by the following fields,
+and needs to be descriptive enough to determine the context related to event occurrence.
+
+1. The `severity`, also referred as `level`, describes the importance of a log entry.
+   Starting from the lower priority to the higher one, the recommended levels are ***debug***, ***info***,
+   ***warning*** (or ***warn***), ***error***, ***fatal***.
+3. Other values as `severity` can be used, but it is recommended to stick to the suggested one.
```

Comment:
Maybe use the one-style format? This means less maintenance and change fatigue when the item order is modified like it happened here. https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md029---ordered-list-item-prefix

Reply:
You're right, I do the that in other projects, but my IDE decides to format them and starts updating values 😢